cluster/gce/gci/configure-helper-common.sh

```diff
@@ -2439,8 +2454,8 @@ function start-workload-controller-manager {
   mkdir -p /etc/srv/kubernetes/workload-controller-manager
   echo "Start workload controller-manager"
   local master_ip=${1:-}  #optional
-  if [[ "${KUBERNETES_RESOURCE_PARTITION:-false}" == "true" ]] || [[ "${KUBERNETES_TENANT_PARTITION:-false}" == "true" ]]; then
-    create-kubeconfig "workload-controller-manager" ${WORKLOAD_CONTROLLER_MANAGER_TOKEN} ${PROXY_RESERVED_IP} "8888" "http"
+  if [[ "${ENABLE_APISERVER_INSECURE_PORT:-false}" == "true" ]]; then
```

Comment:
here as well

Reply:
this section should be ignored. we didn't use workload-controller-manager for a long time. in order to avoid unexpected regression, remove all changing all workload-controller-manager.

Reply:
the workload-controller-manager is a sibling of Kube-controller manager and with very similar code in it. so they should be treated the same way as the Kube-controller-manager, right ? i.e. the same flow should be applied.